### ISSUE TITLE
Cannot find module 'object-assign' (3.1.0)

### ISSUE THREAD

**Issue by medmunds** [jtangelder/sass-loader#178]
I think object-assign needs to be moved from devDependencies into regular dependencies, as of the 3.1.0 release:

```
ERROR in ./src/common/common.scss
Module build failed: Error: Cannot find module 'object-assign'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (.../node_modules/sass-loader/index.js:9:14)
...
```

**Comment by aaron-tay** [comment]
:+1: would love to see this resolved soon

**Comment by hadimichael** [comment]
+1 had a similar issue

**Comment by DanielHeath** [comment]
+1

**Comment by shinygang** [comment]
you should npm install object-assign --save-dev

**Comment by jhnns** [comment]
I'm sorry. `object-assign` was in `devDependencies` :(. Fixed with `3.1.1`

**Comment by dtothefp** [comment]
@jhnns thanks for the quick fix

**Comment by omarinho1** [comment]
> I'm sorry. `object-assign` was in `devDependencies` :(. Fixed with `3.1.1`

danke
